### PR TITLE
Improving documentation related to the JDK 8

### PR DESCRIPTION
**Pull request by danilomunoz**
See the discussion here: caelum/vraptor-java8#15

### CODE REVIEW

File: vraptor-site/content/pt/docs/dependencias-e-pre-requisitos.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 #Dependências e pré-requisitos
 
-O VRaptor 4 depende diretamente do CDI versão 1.1, portanto só funcionará nos servidores que suportem essa versão do CDI ou superior como CDI 1.2. Também é necessário possuir JDK 1.7 ou superior.
+O VRaptor 4 depende diretamente do CDI versão 1.1, portanto só funcionará nos servidores que suportem essa versão do CDI ou superior como CDI 1.2. Também é necessário possuir JDK 1.7 ou superior (veja mais na <a href="#jdk-8">sessão JDK 8</a>).
```
Comment (Turini) [inline vraptor-site/content/pt/docs/dependencias-e-pre-requisitos.html]: what do u think?

>Também é necessário possuir JDK 1.7 ou superior. **Se for usar Java 8, não deixe de conferir este tópico.**

File: vraptor-site/content/en/docs/dependencies-and-prerequisites.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
 #Dependencies and prerequisites
 
 VRaptor 4 depends directly on CDI 1.1, so it will work only in servers that support
-this version of CDI, or higher like CDI 1.2. JDK 1.7 or higher is also mandatory.
+this version of CDI, or higher like CDI 1.2. JDK 1.7 or higher is also mandatory (read more on <a href="#jdk-8">JDK 8 section</a>).
```
Comment (Turini) [inline vraptor-site/content/en/docs/dependencies-and-prerequisites.html]: same here https://github.com/caelum/vraptor4/pull/1066/files#r61165543

**Comment by Turini** [comment]
sounds great, @danilomunoz. I just left I few comments.

**Comment by danilomunoz** [comment]
Done @Turini 
Thank you for your help!

**Comment by Turini** [comment]
awesome!

thank you for spending (investing) time on this, @danilomunoz 
merged

**Pull request merged**